syncmaster/schemas/v1/transfers/file_format.py

```diff
@@ -47,11 +93,14 @@ class XML(BaseModel):
     type: XML_FORMAT
     root_tag: str
     row_tag: str
+    compression: XMLCompression | None = None
```

Comment:
Well, this doesn't look consistent. What to you think about using `None` as `no compression`?

Reply:
I'm not sure if spark-xml supports the `"none"` option. It seems unclear what they mean by `no compression` [here](https://github.com/databricks/spark-xml?ysclid=m49lkp10ot296287479#:~:text=is%20_VALUE.-,compression,-%3A%20compression%20codec%20to). XML will be added soon, and it would be great to check it out

Reply:
If XML integration tests are not ready, then this should be implemented in separated PR

Reply:
> It seems unclear what they mean by no compression [here](https://github.com/databricks/spark-xml?ysclid=m49lkp10ot296287479#:~:text=is%20_VALUE.-,compression,-%3A%20compression%20codec%20to)

This mean that `compression` haven't been passed to options at all:
https://github.com/databricks/spark-xml/blob/ddd1ef573a5318748763fafc974e4f7d8876fd6f/src/main/scala/com/databricks/spark/xml/XmlOptions.scala#L31